pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2433,6 +2433,9 @@ private boolean shouldTopicBeRetained() {
                 return CompletableFuture.allOf(replicationFuture, dedupFuture, persistentPoliciesFuture,
                         preCreateSubscriptionForCompactionIfNeeded());
             });
+        }).exceptionally(ex -> {
+            log.error("update namespace polices : {} error", data, ex);
```

Comment:
Could we add a namespace name to show which namespace update policy fail?

Reply:
Yes, right. added.